src/main/java/com/smartling/marketo/sdk/rest/MarketoApiNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+package com.smartling.marketo.sdk.rest;
+
+import com.smartling.marketo.sdk.MarketoApiException;
+
+public class MarketoApiNotFoundException extends MarketoApiException {
```

Comment:
Exception name confuses. I have read it as some Marketo endpoint is not found. Also, it's not consistent with `RequestLimitExceededException`. So I propose to simplify the name to `AssetNotFoundException` or `ObjectNotFoundException`.

How did you catch it? I see [documentation](https://developers.marketo.com/rest-api/error-codes/) doesn't mention 404.

Reply:
Makes sense, will rename. Have already discovered that Marketo returns empty 200 response when not found even for get by id :/ see updates